### ISSUE TITLE
Landsat-Metadata-CSVs 

### ISSUE THREAD

**Issue by DavidZebris** [olivierhagolle/LANDSAT-Download#10]
Hi,
do you already know this CSV-files with Landsat-metadata ?

http://landsat.usgs.gov/metadatalist.php

You can find an ID and the acqusition-date in this files, so you don't have to calculate the next overpass or to try out all combinations of version and station.
It is possible to calculate the download-links for a given period of time with just a few lines of code from this files.

But on the other hand, this files are quite big and you alway have to download the whole file even if you only want to have the metadata for the latest files only..

**Comment by olivierhagolle** [comment]
Thanks David,
I did not know that page, which could be useful !
But I just downloaded the LANDSAT8 and it is already 208 MBytes only after less than two years !
6 Minutes to download it at home. I think we should not download it each time we want a new product. 
I will see if I have time to integrate it, but keep the current procedure for the most recent products.

**Comment by vascobnunes** [comment]
I'm trying to develop a solution using these csv files. It can save a lot of time if you're looking for cloud free images, for example...

**Comment by olivierhagolle** [comment]
Thanks Vasco, that should be a nice improvement, sorry I have no time to help these days.
Olivier

**Comment by vascobnunes** [comment]
New pull request already with this. Could any of you test it?
